python/pylibcudf/pylibcudf/tests/test_column_from_array.py

```diff
@@ -180,3 +180,18 @@ def test_from_zero_dimensional_array():
         match="shape must be a non-empty tuple",
     ):
         plc.Column.from_array(arr)
+
+
+@pytest.mark.parametrize("np_or_cp_array", [np.array, cp.array])
+@pytest.mark.parametrize(
+    "arr, dtype, expect",
+    [
+        ([], np.int64, pa.array([], type=pa.int64())),
+        ([[], []], np.int32, pa.array([[], []], type=pa.list_(pa.int32()))),
+    ],
+)
+def test_empty_array(np_or_cp_array, arr, dtype, expect):
+    arr = np_or_cp_array(arr, dtype=dtype)
+    col = plc.Column.from_array(arr)
```

Comment:
https://github.com/rapidsai/cudf/issues/19141 uses `plc.Column.from_array_interface`, but these tets use `plc.Column.from_array`. Is that deliberate?

Reply:
Yes, I'm testing both `from_array_interface` and `from_cuda_array_interface`. `from_array` https://github.com/rapidsai/cudf/blob/5d02e2b66237693374cd463957562826f23ede14/python/pylibcudf/pylibcudf/column.pyx#L916-L919 just calls the correct one